python/nwis_client/src/hydrotools/nwis_client/iv.py

```diff
@@ -809,3 +817,17 @@ def flatten_and_stringify(v):
     value_groups = np.array_split(values, n_groups)
 
     return list(map(lambda i: ",".join(i), value_groups))
+
+
+def _create_empty_canonical_df() -> pd.DataFrame:
+    """Returns an empty hydrotools canonical dataframe with correct field datatypes."""
+    cols = {
+        "value_time": pd.Series(dtype="datetime64[ns]"),
```

Comment:
Please see #117 
We could use this as an opportunity to advance that issue or leave it alone. In either case, I think it's a good idea to change `"value_time"` to `self.value_time_label`. I'm open to suggestions.

Reply:
Sorry I am just now getting back to this, with regard to this, I think my vote for how to proceed is to add a temporary keyword only argument to the `_create_empty_canonical_df` function making it possible to alter the `"value_time"` field's name. So, in the case of the implementation in `IVDataService.get`, we can just pass `self.value_time_label` as a parameter in the cases when an empty dataframe is returned. Then, in the future when we get to Update 3 of #117, the temporary `value_time` parameter can be removed from `_create_empty_canonical_df`. Its a little added complexity, but I think it is worth it.

I will make those changes and open a PR if you agree with that approach?

Reply:
> ~~I will make those changes and open a PR if you agree with that approach?~~

I see the changes you made in #135, specifically, d76d75c078c053e4cb4585848433807dd00f535d.